### PR TITLE
fix(mysql): transpile ISNULL to IS NULL

### PR DESCRIPTION
**Pull request by barakalon**
IS NULL seems more standard than the ISNULL function.

I've wrapped it in parenthesis because I'm scared of operator precedence issues.

### CODE REVIEW

File: sqlglot/dialects/dialect.py [hunk 1]
```diff
@@ -760,3 +760,7 @@ def is_parse_json(expression: exp.Expression) -> bool:
     return isinstance(expression, exp.ParseJSON) or (
         isinstance(expression, exp.Cast) and expression.is_type("json")
     )
+
+
+def isnull_to_is_null(args: t.List) -> exp.Expression:
+    return exp.Paren(this=exp.Paren(this=args[0]).is_(exp.null()))
```
Comment (tobymao) [inline sqlglot/dialects/dialect.py:766]: seq_get(args, 0)

Comment (tobymao) [reply sqlglot/dialects/dialect.py:766]: exp.paren()

Comment (GeorgeSittas) [inline sqlglot/dialects/dialect.py:766]: I think the inner parentheses can be omitted? Isn't `IS` non-associative?

Comment (barakalon) [reply sqlglot/dialects/dialect.py]: ah sure

**Review by tobymao — APPROVED** [review]

**Pull request merged**